pom.xml

```diff
@@ -433,6 +434,12 @@
                 <version>${jwt.version}</version>
                 <scope>runtime</scope>
             </dependency>
+
+            <dependency>
+                <groupId>org.influxdb</groupId>
+                <artifactId>influxdb-java</artifactId>
+                <version>${influxdb.version}</version>
+            </dependency>
```

Comment:
Connector-specific dependencies are now managed under seatunnel-connectors-v2/connector-influxdb/pom.xml

Reply:
> Connector-specific dependencies are now managed under seatunnel-connectors-v2/connector-influxdb/pom.xml

ok, i have fixed in new commit